src/app/client/src/app/modules/core/components/content-player-metadata/content-player-metadata.component.ts

```diff
@@ -42,6 +44,21 @@ export class ContentPlayerMetadataComponent implements OnInit, OnDestroy {
     });
   }
 
+  showContentCredits () {
+    const contentCredits = _.get(this.metadata, 'content-credits');
+    this.contentCreditsData = { contributors: '', creators: ''};
+    if (contentCredits && contentCredits.length) {
+      this.contentCreditsData.contributors = _.map(contentCredits, 'name').toString();
```

Comment:
**_.map(contentCredits, 'name').toString();** should be **_.toSting( _.map(contentCredits, 'name'));**